jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/redhatpolicies/misc/RedhatNotScanningClusterPolicy.java

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+ *  Copyright 2023 Paladin Cloud, Inc. or its affiliates. All Rights Reserved.
```

Comment:
oh, it's already 2024 :)

Reply:
changed

Reply:
@akashpaladin you fix one and then added another one with 2023...